openfecli/commands/rbfe_campaigner.py

```diff
@@ -10,68 +10,67 @@
 
 """
 import json
-import pathlib
 import click
+from openfecli.utils import write
 from openfecli import OFECommandPlugin
-from openfecli.parameters import MOL, MAPPER, OUTPUT_FILE_AND_EXT
-from openfecli.parameters.output import ensure_file_does_not_exist
+from openfecli.parameters import MOL, PROTEIN, MAPPER, OUTPUT_FILE_AND_EXT
 
 @click.command(
-    'plan_rbfe',
+    'rbfe_campaigner',
```

Comment:
may I suggest not using "campaigner" here, possibly just "campaign" might make more sense

Reply:
good point!